### PR TITLE
proposal: AuditorYearsOfExperience

### PR DESCRIPTION
**Pull request by corymosiman12**
- added proposal
- updated schema with proposed implementation

### CODE REVIEW

**Review by aviveiros11 — APPROVED** [review]
This looks good to me!

File: BuildingSync.xsd [hunk 1]
```diff
@@ -6091,6 +6091,18 @@
                       <xs:documentation>Type of certification held by an auditor team member.</xs:documentation>
                     </xs:annotation>
                   </xs:element>
+                  <xs:element name="AuditorYearsOfExperience" minOccurs="0">
+                    <xs:annotation>
+                      <xs:documentation>The number of years the energy auditor has been conducting audits professionally.</xs:documentation>
+                    </xs:annotation>
+                    <xs:complexType>
+                      <xs:simpleContent>
+                        <xs:extension base="xs:integer">
```
Comment (macintoshpie) [inline BuildingSync.xsd:6100]: Should we use `nonNegativeInteger` type here?
http://books.xmlschemata.org/relaxng/ch19-77247.html

Comment (JieXiong9119) [reply BuildingSync.xsd:6100]: Is it something we can modify here?

Comment (corymosiman12) [reply BuildingSync.xsd]: addressed f0cc13c

**Comment by JieXiong9119** [comment]
I think it's cool to go.

**Review by JieXiong9119 — APPROVED** [review]

**Review by yan130 — APPROVED** [review]

**Review by nllong — APPROVED** [review]
🤘

**Pull request merged**